src/GeneralStateTestsFiller/EIPTests/stEOF/stEIP3540/CreateTransactionEOF1Filler.yml

```diff
@@ -59,16 +59,6 @@ CreateTransactionEOF1:
       # deployed code section: 'PUSH1 0 STOP' data section: 'aabbccdd'
       - ':label eof1_initcode_with_data_deploying_valid_eof1_with_data :raw 0xef0001010029020002007fef000101000302000400600000aabbccdd00000000000000000000000000000060005260116000f3eeff'
 
-      # EOF initcode => legacy code
```

Comment:
We still need to have a test for this

> For a create transaction, if _initcode_ or _code_ is invalid, the contract creation results in an exceptional abort. Such a transaction is valid and may be included in a block.

It must check that transaction was valid, but nothing was deployed